js/directDebitInformation.js

```diff
@@ -0,0 +1,51 @@
+CRM.$(function ($) {
+
+  renderMandateCustomData();
+
+  function renderMandateCustomData() {
+    var urlData = CRM.urlData || {};
+
+    var url = CRM.url(
+      'civicrm/contact/view/cd',
+      {
+        reset: '1',
+        gid: urlData.gid,
+        cid: urlData.cid,
+        recId: urlData.recId,
+        multiRecordDisplay: 'single',
+        mode: 'view'
+      }
+    );
+
```

Comment:
DD-6 point 3:  we have to inject custom group direct debit information right below "View Recurring Payment" page. And for this purpose we always use JS function loadPage(), with 'modal = false' but our custom group is reserved and view only. Maybe you know other methods for injecting custom groups or some tips how to find proper URL.  We hardcoded our block of code just to clarify our intention. We also try to use CiviCRM REST API, as  you suggested, but it was't successful.